onnxruntime/core/platform/windows/env.cc

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 
 #include <Shlwapi.h>
 #include <Windows.h>
+#include <shellapi.h>
```

Comment:
Who need shell api? I guess WinML team don't want it.

Reply:
SHFileOperation() in DeleteFolder(). Who can confirm whether it's undesirable, and if so what's a good replacement?

Reply:
Should only use APIs allowed by UWP. See if this can be reimplemented with different APIs.

---
In reply to: [393223052](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r393223052) [](ancestors = 393223052)